Update installation instructions for swift-format (#762)
NSHipster/articles: 2019-05-20-swift-format.md

#### Installation

> Install using homebrew:

Reviewer: ```suggestion
You can install using [Homebrew](https://brew.sh) with the following command:
```

```terminal
$ brew install swift-format
```

> You may also clone and build yourself over at 

Reviewer: ```suggestion
Alternatively, you can clone its [source repository](https://github.com/apple/swift-format) and build it yourself.
```
